routes/views/director_profile.js

```diff
@@ -16,9 +16,33 @@ exports = module.exports = (req, res) => {
 
       if (err) return next(err);
 
-      locals.director = director;
+      // find out which fields in this model are translatable fields
+      // don't want to hardcode this, as I would like to export this function and use it across all routes
+      // also translatable key names have pattern name___LOCALE << yes, 3 underscores!!
+      const translatableFields = Object.keys(director._doc)
+        .filter(key => key.includes('___'))
```

Comment:
You're using two approaches for searching here;

Before on line 38 you using a strong regular expression;
```javascript
b.replace(/\_\_.+$/
```
Here your using loose string checking
```javascript
.filter(key => key.includes('___'))
```

You could store an expression matcher in the higher scope;
```javascript
const translatableKeyReg = /\_{3}.+$/;
```
Then use it with;
```javascript
.filter(key => translatableKeyReg.test(key))

...

[b.replace(translatableKeyReg, '')

```